src/pages/Components/RatingStars/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Rating Stars'
+order: 1
+---
+
+> Rating stars is a pattern you'll find in almost every website. It has two main usages. First, it allows the user to give feedback on the current experience with a product or a service. Second, it allows a brand to provide some informations on other user's opinion on a product or a service to make his choice easier.
+
+<preview path="src/pages/Components/RatingStars/previews/stars-input-basic" nude=true></preview>
+<preview path="src/pages/Components/RatingStars/previews/stars-result-basic" nude=true></preview>
+
+## Variations
+
+<preview path="src/pages/Components/RatingStars/previews/stars-result-all-states" nude=true></preview>
+
+We have separated rating stars into `mc-stars-input` and `mc-stars-results`.
```

Comment:
it's a developer term, we need to use a generic term. The usage page can be visited by designers, developers and product owners.

Reply:
Names should be the same, and sketch libraries should reflect the naming. 
We needed to have 2 patterns, one for the display, one as an input for notation. 

At some point if we want to make changes of design in one and not the other (for example to create a more distinct difference between a clickable one) we may create a breaking change in the design with all designers required to make changes in their designs. 

If we are consistant, a change in one and not the other will update designs files without designer intervention. 

I think this is really important. 

But I don't think we need the prefix (mc-) just `stars-input` and `stars-results`

Reply:
I talked with JB, for the moment there is no reason to create a visual difference between the two cases. Moreover we will have a difficult file to understand for the designers.